stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py

```diff
@@ -58,8 +58,13 @@ def update_item(self, model: stac_types.Item, **kwargs) -> stac_types.Item:
             query = session.query(self.item_table).filter(
                 self.item_table.id == model["id"]
             )
+            query = session.query(self.item_table).filter(
```

Comment:
This definition of `query` seems to override the next line. Perhaps use `query` instead of `session.query(...)` here.